### PR TITLE
error when supplied a trained recipe

### LINKED ISSUE #179: Do not allow fitted recipes
Another thing that I've seen is that people prep a recipe then use `add_recipe()`. Can we error when that happens and add something to the man file for `add_recipe()`? 

``` r
library(tidymodels)

training <- mtcars[1:20, ]
testing <- mtcars[21:32, ]

model <- linear_reg() %>%
  set_engine("lm")

workflow <- workflow() %>%
  add_model(model)

recipe <- recipe(mpg ~ cyl + disp, training) %>%
  step_log(disp) %>% 
  prep()

map_lgl(recipe$steps, is_trained)
#> [1] TRUE

workflow <- add_recipe(workflow, recipe)

fit_workflow <- fit(workflow, training)
```

<sup>Created on 2022-09-21 by the [reprex package](https://reprex.tidyverse.org) (v2.0.1)</sup>

### PR DESCRIPTION
**Pull request by simonpcouch**
Closes #179.

### CODE REVIEW

File: R/pre-action-recipe.R [hunk 1]
```diff
@@ -19,7 +19,9 @@
 #'
 #' @param x A workflow
 #'
-#' @param recipe A recipe created using [recipes::recipe()]
+#' @param recipe A recipe created using [recipes::recipe()]. The recipe
+#'    should have been trained already with [recipes::prep()]; workflows
+#'    will handle training internally.
```
Comment (EmilHvitfeldt) [inline R/pre-action-recipe.R:24]: I think we need a "not" in here somewhere 😄

**Review by EmilHvitfeldt — APPROVED** [review]
One important change suggestion, otherwise super smooth and condensed PR!

File: R/pre-action-recipe.R (hunk 1 above)
Comment (simonpcouch) [reply R/pre-action-recipe.R:24]: ha, you're right!

Comment (simonpcouch) [reply R/pre-action-recipe.R]: Addressed in [30b5fc7](https://github.com/tidymodels/workflows/pull/202/commits/30b5fc7b69e0e499b7c2bd6437a00dbf8a1f8f3d).🪱

**Pull request merged**